2_globus.ipynb

```diff
@@ -20,17 +20,6 @@
     "* Explain how to find the Guest Collection"
```

Comment:
<p>I've opened issue #11 to make sure files like this are persistent</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cheapandfair/cheapandfair-gateways-2024/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>